vips/vip-192/commands-testnet.ts

```diff
@@ -0,0 +1,292 @@
+const XVS_VAULT_IMPL = "0x37BB5aBBDfFD81372fBeA830B9d5b38B98551c06";
+const PRIME_LIQUIDITY_PROVIDER = "0xce20cACeF98DC03b2e30cD63b7B56B018d171E9c";
+const PRIME = "0xb13Ea8C39594449B2AB5555769580BDB23f5E2Cf";
+
+const ETH = "0x98f7A83361F7Ac8765CcEBAB1425da6b341958a7";
+const BTC = "0xA808e341e8e723DC6BA0Bb5204Bafc2330d7B8e4";
+const USDC = "0x16227D60f7a0e586C66B005219dfc887D13C9531";
+const USDT = "0xA11c8D9DC9b66E209Ef60F0C8D969D3CD988782c";
+
+const vETH = "0x162D005F0Fff510E54958Cfc5CF32A3180A84aab";
+const vBTC = "0xb6e9322C49FD75a367Fcb17B0Fcd62C5070EbCBe";
+const vUSDC = "0xD5C4C2e2facBEB59D0216D0595d63FcDc6F9A1a7";
+const vUSDT = "0xb7526572FFE56AB9D7489838Bf2E18e3323b441A";
+
+const PREVIOUSLY_STAKED_USER = "0x2Ce1d0ffD7E869D9DF33e28552b12DdDed326706";
+const STAKED_AT = "1698270716";
+
+const cutParams = params;
+
+export default [
+  {
+    target: UNITROLLER,
+    signature: "diamondCut((address,uint8,bytes4[])[])",
+    params: [cutParams],
+  },
+  {
+    target: XVS_VAULT_PROXY,
+    signature: "_setPendingImplementation(address)",
+    params: [XVS_VAULT_IMPL],
+  },
+  {
+    target: XVS_VAULT_IMPL,
+    signature: "_become(address)",
+    params: [XVS_VAULT_PROXY],
+  },
+  {
+    target: PRIME_LIQUIDITY_PROVIDER,
+    signature: "acceptOwnership()",
+    params: [],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "setTokensDistributionSpeed(address[],uint256[])", NORMAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", NORMAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", NORMAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "setTokensDistributionSpeed(address[],uint256[])", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "setTokensDistributionSpeed(address[],uint256[])", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", CRITICAL_TIMELOCK],
+  },
+  {
```

Comment:
Grant Guardian wallet to execute `pauseFundsTransfer()` and `resumeFundsTransfer()` on the PrimeLiquidityProvider contract